configure.win

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+
+RWITHPYTHON_PYTHON_VERSION=3.5
+RWITHPYTHON_PYTHON_VERSION_EXTENDED=3.5.2
+
+if [ $R_ARCH = "/x64" ]
+then
+	EMBEDDED_ZIP=https://www.python.org/ftp/python/${RWITHPYTHON_PYTHON_VERSION_EXTENDED}/python-${RWITHPYTHON_PYTHON_VERSION_EXTENDED}-embed-amd64.zip
+else
+	EMBEDDED_ZIP=https://www.python.org/ftp/python/${RWITHPYTHON_PYTHON_VERSION_EXTENDED}/python-${RWITHPYTHON_PYTHON_VERSION_EXTENDED}-embed-win32.zip
+fi
+
+# This creates a folder /pythonLibs in the installed package.
+# Note:  You can't create /libs which is reserved for the R package installer
+PYTHON_LIB_PATH=inst/pythonLibs
+mkdir -p ${PYTHON_LIB_PATH}
+cd ${PYTHON_LIB_PATH}
+wget -O python-${RWITHPYTHON_PYTHON_VERSION_EXTENDED}-embed.zip $EMBEDDED_ZIP
+if [ "$?" -ne "0" ]; then
+  echo "Failed to download $EMBEDDED_ZIP"
+  exit 1
+fi
+unzip python-${RWITHPYTHON_PYTHON_VERSION_EXTENDED}-embed.zip
+
+PIP_INSTALL_SCRIPT=get-pip.py

```

Comment:
verify that installing python.exe doesn't collide in any way with the system's current copy.